---
layout: default
title: Yaoyu Yang | Research
---
<div class="container">
<h1>Research Projects</h1>
<div class="col-md-4" style='margin-top:20px'>
  <div class="imgAbt">
    <img style = 'margin: auto' width="260" height="260" class="img-responsive img-rounded" src="/images/bistable_switch_paper_vertical.png" />
  </div>
</div>
<div class="col-md-8" style='margin-top:20px'>
  <h4>Synthetic Bistability and Differentiation in Yeast</h4>
  <p>Engineered systems that control cellular differentiation and pattern formation are essential for applications like tissue engineering, biomaterial fabrication, and synthetic ecosystems. Synthetic circuits that can take on multiple states have been made to engineer multicellular systems. However, how to use these states to drive interesting cellular behavior remains challenging. I built a cellular differentiation program involving a novel synthetic bistable switch coupled to an antibiotic resistance gene that affects growth in yeast (S. cerevisiae). The switch is composed of a positive feedback loop involving a novel transcription factor and can be switched ON and OFF via two different transient inducer inputs. By further coupling the bistable switch with an antibiotic resistance gene, we obtained a growth differentiation circuit, where yeast cells can be switched to stable HIGH or LOW growth rate states via transient inducer inputs. This work demonstrates a rationally designed and experimentally validated cellular differentiation behavior in yeast.</p>
  <h5>Related work</h5>
  <ul class='pubs'>
    <li> Y. Yang, J. L. Nemhauser, E. Klavins. <b>Synthetic Bistability and Differentiation in Yeast</b>, <i>ACS synthetic biology</i>, 2019. <a href='https://pubs.acs.org/doi/abs/10.1021/acssynbio.8b00524' target="_blank"><span class="glyphicon glyphicon-link"></span></a>
    <li> Y. Yang, E. Klavins. <b>Design and construction of a bistable switch in yeast</b>. Poster and Lightning	Talk, <a href='http://molecular-programming.org/' target="_blank"><i>Molecular Programming Project Workshop</i></a>, San Francisco, USA, 2015.
    </li>
    <li> Y. Yang, E. Klavins. <b>Design and construction of a switchable bistable switch in yeast</b>. Poster, <a href='http://synbioconference.org/2015/seed-2015-program' target="_blank"><i>SEED 2015</i></a>, Boston, USA, 2015.
    </li>
  </ul>
</div>
<div class="col-md-4" style='margin-top:20px'>
  <div class="imgAbt">
    <img style = 'margin: auto' width="260" height="260" class="img-responsive img-rounded" src="/images/oscillator_network.png" />
  </div>
</div>
<div class="col-md-8" style='margin-top:20px'>
  <h4>A plant hormone induced oscillator in yeast</h4>
  <p>Biochemical oscillators are fundamental to understanding cell physiology, and present an ongoing challenge in building synthetic living systems. In synthetic biology, researchers have built a number of oscillators over the last decade resulting in various designs in organisms from E. coli to mammalian cells. However, especially in eukaryotic cells, a robust, tunable, and modular oscillator technology has yet to be demonstrated. In this project, I designed and analyzed a plant hormone induced oscillator in yeast. I developed mathematical model and showed that the oscillator could be achieved by tuning two key parameters. Currently I am working on experimental realizing this oscillator.</p>
  <h5>Related work</h5>
  <ul class='pubs'>
  <li> Y. Yang, E. Klavins. <b>Design and analysis of a plant hormone induced oscillator in yeast</b>. Abstract and talk, <a href='http://www.csh-asia.org/2014meetings/Synthesi.html' target="_blank"><i>CSHA Synthetic Biology</i></a>, Suzhou, China, 2014. <a href='http://www.csh-asia.org/photos/14synthesi/index.htm' target="_blank"><span class="glyphicon glyphicon-link"></span></a>
  </li>
  </ul>
</div>

<div class="col-md-4" style='margin-top:20px'>
  <div class="imgAbt">
    <img style = 'margin: auto' width="260" height="260" class="img-responsive img-rounded" src="/images/aquarium_ecoli.png" />
  </div>
</div>

<div class="col-md-8" style='margin-top:20px'>
  <h4>The Aquarium cloning workflows</h4>
  <p><a href="http://www.aquarium.bio/" target="_blank">Aquarium</a> is a smart laboratory operating system invented in the <a href="http://klavinslab.org" target="_blank">Klavins Lab</a> where researchers can specify how to manipulate items that stored in the inventory using protocols and workflows written in computer codes. Adapted from the standard molecular cloning workflows, I wrote a majority of the codes in Ruby implementing two core molecular cloning workflows: plasmid construction and yeast cloning. These workflows take and batch users' inputs to the workflow and semi-automatically schedule the entire cloning process from ordering primers to Sanger sequencing verifications, resulting a nearly effortless cloning process in Aquarium. This technology also led to the startup <a href="http://www.aquarium.bio/" target="_blank">Aquarium</a> where I worked as an engineer to build out the 21st century laboratory operating system, enabling reproducible and transferable workflows in biology labs and more.<p>
  <h5>Related work</h5>
  <ul class='pubs'>
    <li> J. Vrana, O. de Lange, Y. Yang, G. Newman, A. Saleem, A. Miller, C. Cordray, S. Halabiya, M. Parks, E. Lopez, S. Goldberg, B. Keller, D. Strickland, E. Klavins <b>Aquarium: open-source laboratory software for design, execution and data management</b>, <i>Synthetic Biology</i>, 2021. <a href='https://academic.oup.com/synbio/article/6/1/ysab006/6124325?login=true' target="_blank"><span class="glyphicon glyphicon-link"></span></a>
    </li>
    <li> Aquarium Lab in the Cloud. <b>Engineering research projects haul in CoMotion Innovation Fund dollars</b>. News, University of Washington, 2015. <a href='https://www.engr.washington.edu/news/comotion-innovation-fund-2015' target="_blank"><span class="glyphicon glyphicon-link"></span></a>
    </li>
  </ul>
</div>

<div class="col-md-4" style='margin-top:20px'>
  <div class="imgAbt">
    <img style = 'margin: auto' width="260" height="260" class="img-responsive img-rounded" src="/images/frequency.png" />
  </div>
</div>

<div class="col-md-8" style='margin-top:20px'>
  <h4>Frequency response for a synthetic auxin signaling pathway in yeast</h4>
  <p>I developed an experimental process in the <a href="http://klavinslab.org" target="_blank">Klavins Lab</a> to interrogate frequency response for a synthetic auxin signaling pathway in yeast. The approach was using a microfluidic device called <a href="http://www.emdmillipore.com/US/en/life-science-research/cell-culture-systems/cellASIC-live-cell-analysis/microfluidic-plates/68eb.qB.wfkAAAFBWmVb3.sJ,nav" target="_blank">CellASIC</a> to grow the yeast cell, where one can control the system precisely with different frequencies of auxin inputs and observe using fluorescence microscopy over time. I wrote Matlab based software to do image processing on the time course fluorescence images, which resulted in quantified time-course fluorescence data on the cells. Using the data, I generated a <a href="http://en.wikipedia.org/wiki/Bode_plot" target="_blank">Bode plot</a> describing the frequency response of signaling pathway. The result showed that this pathway works as a low-pass filter.</p>
  <h5>Related work</h5>
  <ul class='pubs'>
  <li> Y. Yang, E. Klavins. <b>Frequency domain signal processing in biological system</b>. Poster and Lightning	Talk, <a href='http://molecular-programming.org/workshops/one.html' target="_blank"><i>Molecular Programming Project Workshop</i></a>, Oxnard, CA, USA, 2013.
  </li>
  </ul>
</div>

